pkg/shell/hosts_dialog.jsx

```diff
@@ -463,21 +464,29 @@ class HostKey extends React.Component {
             body = <>
                 <Alert variant='danger' isInline title={_("Changed keys are often the result of an operating system reinstallation. However, an unexpected change may indicate a third-party attempt to intercept your connection.")} />
                 <p>{_("To ensure that your connection is not intercepted by a malicious third-party, please verify the host key fingerprint:")}</p>
-                <pre className="hostkey-fingerprint">{fp}</pre>
```

Comment:
You removed this and some other classes, but I see these are used in the tests. 

test/verify/check-shell-multi-machine:                self.assertEqual(b.text('#hostkey-fingerprint'), fp)
test/verify/check-shell-multi-machine:                self.assertTrue(b.val('#hostkey-fingerprint'))


I suggest you keep the classes for the tests to work. 

If these is CSS that you want to drop thats' assosiated with the class, we need to check each case, and drop it if no longer relevant and adjust the classname like here, so that's affecting only the non-changed files:

pkg/static/login.css:#hostkey-fingerprint {
pkg/static/login.html:          <pre id="hostkey-fingerprint"></pre>